src/app/components/Metadata/index.test.tsx

```diff
@@ -1013,11 +1013,11 @@ describe('Snapshot', () => {
         bbcOrigin={dotComOrigin}
         platform="canonical"
         id={null}
-        pageType={MEDIA_PAGE}
+        pageType={LIVE_RADIO_PAGE}
         pathname="/korean/bbc_korean_radio/liveradio"
-        title={liveRadioPageData.promo.name}
-        lang={liveRadioPageData.metadata.language}
-        description={liveRadioPageData.promo.summary}
+        title={liveRadioPageData.data.name}
+        lang={liveRadioPageData.data.language}
+        description={liveRadioPageData.data.summary}
         openGraphType="website"
       />,
```

Comment:
Can we add `hasAmpPage=false` as live radio no longer supports amp (this will result in an updated snapshot)